packages/next/build/webpack/loaders/next-flight-client-loader/index.ts

```diff
@@ -16,12 +16,14 @@ function containsPath(parent: string, child: string) {
 
 export default async function transformSource(
   this: any,
-  source: string
-): Promise<string> {
+  source: string,
+  ...rest: any
```

Comment:
```suggestion
  map: any
```

Let's just name the parameter. The parameters are stable in this and `callback` is not guaranteed to take more than `output, map`